documentation/Bookdown.md

```diff
@@ -0,0 +1,39 @@
+## Bookdown installation 
+
+To use Bookdown you first need to install:
+* [R](https://www.r-project.org/) programming language. 
+* The development environment [RStudio](https://www.rstudio.com/).
+* Add the R packages [Bookdown](https://github.com/rstudio/bookdown) and [JuliaCall]( https://github.com/Non-Contradiction/JuliaCall).
+    
+    NOTE: For the time being, JuliaCall is having an [issue](https://github.com/Non-Contradiction/JuliaCall/issues/164) with the Julia 1.6 version. So you will need to set Julia version v1.5.4 or older as the default one.
+
+## Workflow
+
+1. Make all the modifications to the chapter you are working on in the .Rmd file.
+
+2. Once you finish making all the modifications, run all the code cells to make sure you don't get any error.
+
+3. Since Data science in Julia for hackers has chapters that involve a lot of computation we render each chapter separately.
```

Comment:
It's standard to write book titles in italics.
Also, in English, every word of the title is capitalized, like so: Data Science in Julia for Hackers